### DISCUSSION TITLE
WIKI:Dogecoin Core Dev for Beginners Like Me

### DISCUSSION THREAD

**Discussion by tosufever** [veryengineer/collab#14] [General]
just wanted to say thanks for this one DogestMaximus! i'm not trying to dive into core, but this helped with "i don't know how to properly use github."  i've only ever used github like thingiverse... find the something close to what you need, download edit & deoloy.  

i might build a vm and roll through this just to do it. much wow.

https://github.com/veryengineer/collab/wiki/Dogecoin-Core-Dev-for-Beginners-Like-Me

**Comment by DogestMaximus** [comment]
I'm still learning myself but [xanimo](https://github.com/xanimo) has taken the time to teach me so truly the credit should go to him.
